### ISSUE TITLE
Inlining

### ISSUE THREAD

**Issue by davidfwall** [leemunroe/htmlemail#6]
Hi there — more of a question or query than an issue. I'm very happy with the templates which have proven useful.

What tools / process did HTMLEmail team use to create the inlined versions of the mails? I've not been able to recreate the transition from non-inlined to inlined without big layout changes, even when the changes that I've made are minor (eg background colour). Was the inlining done manually in full or in part?

thanks!

**Comment by leemunroe** [comment]
Hey @davidfwall. The process used to create these was a modified version of this [Grunt workflow](https://github.com/leemunroe/grunt-email-workflow). This uses Juice to do the inlining, and then there was some manual clean up done as well.

Curious to hear (and see) the issues you're having to see if I can help. What inliner are you using? What specific issues are you having?

Also, is having access to the workflow's source files something you would find useful? I've been meaning to look into packaging it up for more advanced developers like yourself too.

**Comment by davidfwall** [comment]
Thanks Lee for getting back! 

I was sending with Mailchimp, so I tried their own one, then Premailer (I also tried the Nokogiri HTML parser) and then Putsmail’s inliner. I was able to get around the issue by isolating the offending lines and copying in from the pre-inlined versions. 

The issues that I was having were in the multi column layouts in the newsletter style templates. The starting point was a hacked up version of the newsletter template. It's quite possible that I broke something along the way myself. 

That grunt workflow looks very useful and at just about the limits of my ability 😬 — I’ll run through it with the files that I had and see what the outcome is like.

thanks
